src/main/java/com/radixdlt/crypto/keystore/Crypto.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *
+ *  * (C) Copyright 2020 Radix DLT Ltd
+ *  *
+ *  * Radix DLT Ltd licenses this file to you under the Apache License,
+ *  * Version 2.0 (the "License"); you may not use this file except in
+ *  * compliance with the License.  You may obtain a copy of the
+ *  * License at
+ *  *
+ *  *  http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  * either express or implied.  See the License for the specific
+ *  * language governing permissions and limitations under the License.
+ *
+ */
+
+package com.radixdlt.crypto.keystore;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Crypto {
+    @SerializedName("cipher")
+    @Expose
+    private String cipher;
+    @SerializedName("cipherparams")
```

Comment:
Spacing between individual members would make this easier to read

Reply:
Agreed! This was just copied over, adding newlines after each instance variable 👍